packages/core/src/components/popover/popover.tsx

```diff
@@ -381,10 +381,13 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
         if (props.isModal && props.interactionKind !== PopoverInteractionKind.CLICK) {
             throw new Error(Errors.POPOVER_MODAL_INTERACTION);
         }
-        try {
-            React.Children.only(props.children);
-        } catch (e) {
-            throw new Error(Errors.POPOVER_ONE_CHILD);
+        if (typeof props.children === "object") {
```

Comment:
i'm not sure what changed but i was getting errors for tooltips with string targets (a very common use case, I imagine). `React.Children.only` asserts that it must have a single _React element_ child, which does not include strings.

this checks first that children is not a basic type before testing the assertion.